tvb_library/tvb/basic/neotraits/info.py

```diff
@@ -33,6 +33,7 @@
 import numpy
 import typing
 
+from docrepr import sphinxify
```

Comment:
We should do a `try ... except ImportError` as before, to avoid breaking working envs

Reply:
I dropped the docrepr commit